integrationtest/wildfly/pom.xml

```diff
@@ -30,6 +30,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <byteman.agent.path>${org.jboss.byteman:byteman:jar}</byteman.agent.path>
         <byteman.script.path>${basedir}${file.separator}src${file.separator}test${file.separator}resources${file.separator}disablejpadapters.btm</byteman.script.path>
+        <hibernate.search.module.slot>main</hibernate.search.module.slot>
```

Comment:
You're right, thanks for noticing. Ok I'll change this, my point about the user having to write long names does not apply here anyway.